docs/README.md

```diff
@@ -1,5 +1,11 @@
 #CAPx Documentation
 
+##Web server system requirements
+
+- Linux xxx
```

Comment:
There is also a PHP5.4 compatible version of this module with an older version of Guzzle on the `php54-2.x` branch. We should link to that as well so that users have a choice. PHP 5.4 is at end of life so hopefully this wont be an issue for long.